### DISCUSSION TITLE
Submission

### DISCUSSION THREAD

**Discussion by yhl-54** [Part-II-Computational-Physics/.github#19] [General]
I tried editing the file on github and committed the changes, however now the it displays invalid notebook and I cannot go back to edit the file.

**Comment by AustenLamacraft** [comment]
Looks fine now, right?

**Comment by NathanMagnan** [comment]
If not, this seem to be (or have been) a common issue, you can find a workaround here:
https://github.com/jupyter/notebook/issues/6507
